### DISCUSSION TITLE
Auto loading a ,json file into rdio

### DISCUSSION THREAD

**Discussion by Nirbles** [chuot/rdio-scanner#431] [Q&A]
Hello all. I have rdio running and saved my configuration to a .json file which it places into my downloads. My question is each time I launch rdio I have to manually import the json file using rdio. Is there a way to automate this process? Thank you.

**Comment by cr8tor456** [comment]
Did you ever figure this out?
Ive been googling and searching this forum for hours now and cant not figure out how to do this.
I cant believe this is not a default feature? Who would want to reconfigured a system every time you have to restart or what not?
I am missing something right?
Please tell me im an idiot? haha

**Comment by Nirbles** [comment]
No I never did figure this out...still doing manually.

**Reply by cr8tor456** [reply]
I automated it a bit with a batch file.
Then i dropped a shortcut into the start location so when windows reboots it automatically starts and loads. 
Works for me for now.  I get the feeling the developer just hasnt worked this in yet, and all good, hopefully he does someday.
Make sure you first export and then put the json file into the same folder as rdio scanner.

Batch file contents:
start rdio-scanner.exe
rdio-scanner.exe -cmd login +password <password>
rdio-scanner.exe -cmd config-set +in rdio-scanner.json

**Reply by Nirbles** [reply]
Thanks...
so password would be my password for admin, and i copied my *json file into radioscanner folder so should work?

This is my current batch file:
C:\Users\timothy\rdio-scanner>rdio-scanner ^
-listen :80 ^
-ssl_cert_file mycert.crt ^
-ssl_key_file meykey.key ^
-ssl_listen :443

I would just add those lines to it right?
Thanks!

**Reply by cr8tor456** [reply]
Well, im not sure how to make that work.
The first line on mine begins with start as that start command tells it to open the app in another window.
If it opens in the existing window, it never runs the other lines as it never "finishes" the first line to be able to continue.
So the first line if you can turn that into a single line type start command, or put it in another batch file to call from the first batch file that might work.

**Comment by Nirbles** [comment]
So I ran the following:
start c:\users\timothy\rdio-scanner\rdio-scanner.exe
rdio-scanner.exe -cmd login +password
rdio-scanner.exe -cmd config-set +in rdio-scanner.json

if i didn't put the path in it couldn't find rdio-scanner.exe
Think it is only running the first line
I do have rdio-scanner.json in the rdio-scanner folder.

Thank you
